components/loadercontainer/overview.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Overview
+page_title: LoaderContainer Overview
+description: Overview of the LoaderContainer for Blazor.
+slug: loadercontainer-overview
+tags: telerik,blazor,loader,container,loadercontainer,overview
+published: True
+position: 0
+---
+
+# LoaderContainer Overview
+
+This article provides information about the LoaderContainer for Blazor and its core features.
+
+The LoaderContainer provides an animated indicator, a panel, and an overlay that can be used when the application is performing a time-consuming operation, for example, loading data.
+
+## Basic LoaderContainer
+
+To add a Telerik LoaderContainer to your Blazor application, use the `<TelerikLoaderContainer>` tag and show it when needed by your app by using its `Visible` paramter. You can also control its [visual appearance]({%slug loadercontainer-appearance%}) through parameters.
+
+![](images/loadercontainer-overview-example.gif)
```

Comment:
The screenshot could be less tall by probably 100-200px. Also, consider moving it after the code snippet.

Reply:
I did record a smaller gif, but I would like it to stay above the snippet so that the customers can get a quick glance at what they are getting with the component. It this okay?